### PR TITLE
refactor(@schematics/angular): Do not include Protractor test in new project

### PR DESCRIPTION
**Pull request by kyliau**
Do not include Protractor for new projects since we do not have a concrete plan for Protractor yet.

Closes https://github.com/angular/angular-cli/issues/18844

### CODE REVIEW

File: packages/angular/cli/commands/e2e-impl.ts [hunk 1]
```diff
@@ -14,8 +14,14 @@ import { Schema as E2eCommandSchema } from './e2e';
 export class E2eCommand extends ArchitectCommand<E2eCommandSchema> {
   public readonly target = 'e2e';
   public readonly multiTarget = true;
+  public readonly missingTargetError = `
+Cannot find "e2e" target for the specified project.
 
-  public async run(options: E2eCommandSchema & Arguments) {
-    return this.runArchitectTarget(options);
+You should add a package that implements e2e capabilities.`;
```
Comment (alan-agius4) [inline packages/angular/cli/commands/e2e-impl.ts:20]: You should add a package that implements end-to-end testing capabilities.

**Review by alan-agius4 — APPROVED** [review]
One optional NIT, otherwise LGTM

File: packages/angular/cli/commands/e2e-impl.ts [hunk 2]
```diff
@@ -14,8 +14,19 @@ import { Schema as E2eCommandSchema } from './e2e';
...
   public readonly target = 'e2e';
   public readonly multiTarget = true;
+  public readonly missingTargetError = `
+Cannot find "e2e" target for the specified project.
 
-  public async run(options: E2eCommandSchema & Arguments) {
-    return this.runArchitectTarget(options);
+You should add a package that implements e2e capabilities.
+
+Here are some options that are available:
```
Comment (alan-agius4) [inline packages/angular/cli/commands/e2e-impl.ts:22]: Would be nice to keep the same format as the other builders ie: https://github.com/angular/angular-cli/blob/c3cc50e6b95fbbd814d752eec808ea0a0ef411c0/packages/angular/cli/commands/deploy-impl.ts#L18

File: packages/angular/cli/commands/e2e-impl.ts [hunk 3]
```diff
@@ -14,8 +14,19 @@ import { Schema as E2eCommandSchema } from './e2e';
...
+  1. TestCafe (https://testcafe.io/documentation/402793/recipes/integrations/use-angular-cli-builder)
+  2. WebdriverIO (https://webdriver.io/docs/protractor-migration/)
```
Comment (alan-agius4) [inline packages/angular/cli/commands/e2e-impl.ts:24]: Should we mentioned this? Since it cannot be used with this command?

Comment (clydin) [reply packages/angular/cli/commands/e2e-impl.ts:24]: maybe like here: https://github.com/webdriverio/webdriverio-schematics

File: packages/angular/cli/commands/e2e-impl.ts (hunk 1 above)
Comment (kyliau) [reply packages/angular/cli/commands/e2e-impl.ts]: done

File: packages/angular/cli/commands/e2e-impl.ts (hunk 2 above)
Comment (kyliau) [reply packages/angular/cli/commands/e2e-impl.ts]: Mentioned `ng add` for WebdriverIO, removed TestCafe for now

File: packages/angular/cli/commands/e2e-impl.ts (hunk 3 above)
Comment (kyliau) [reply packages/angular/cli/commands/e2e-impl.ts]: Removed TestCafe for now, opened https://github.com/politie/angular-testcafe/issues/63 to ask them to add schematic.
Mentioned `ng add` for WebdriverIO.

**Review by christian-bromann — APPROVED** [review]
👍

**Review by clydin — APPROVED** [review]

**Pull request merged**